### PR TITLE
use PHP 5.5, remove $self = $this

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

**Comment by samsonasik** [comment]
rebased

File: src/AbstractPluginManager.php [hunk 1]
```diff
@@ -63,9 +63,9 @@ public function __construct(ConfigInterface $configuration = null)
     {
         parent::__construct($configuration);
         $self = $this;
```
Comment (sandrokeil) [inline src/AbstractPluginManager.php]: Why you don't remove the line `$self = $this`?

Comment (samsonasik) [inline src/AbstractPluginManager.php]: done ;)

**Comment by weierophinney** [comment]
Moving to 2.6.0, as I want to do that release now.

**Pull request closed without merging**